app/views/whitehall_migration/document.html.erb

```diff
@@ -1,2 +1,27 @@
-<h1>Document</h1>
-<%= @document.id %>
+<% content_for :back_link, render_back_link(href: whitehall_migration_documents_path(migration_id: @document_import.whitehall_migration, state: @document_import.state)) %>
+<% content_for :title, "Document Import: #{@document_import.id}" %>
+<%= render "govuk_publishing_components/components/summary_list", {
+  items: [
+    {
+      field: "ID",
+      value: @document_import.id
+    },
+    {
+      field: "Created at",
+      value: format_time_on_date(@document_import.created_at)
+    },
+    {
+      field: "Updated at",
+      value: @document_import.whitehall_migration.updated_at.nil? ? "unfinished" : format_time_on_date(@document_import.whitehall_migration.updated_at)
```

Comment:
I think we should the document imports updated_at rather than the migrations one, I don't think we need a conditional for unfinished here (the current condition here would never be nil once persisted to the db)